test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -704,6 +705,33 @@ def forward(self, x):
         FileCheck().check_count("aten::add", 1, exactly=True).run(mqt.m2.graph)
         FileCheck().check_count("quantized::add", 0, exactly=True).run(mqt.m2.graph)
 
+    def test_module_list(self):
```

Comment:
nit: just to confirm, do we have a test for Sequential as well? also for completeness do we want to add support for ModuleDict?

Reply:
yes, sequential is already tested.  ModuleDict could be added in a future PR, it's just not needed for the current model we are working on.